### ISSUE TITLE
aliases filter question

### ISSUE THREAD

**Issue by Feng-YiJing-Dao** [Stardusten/ob-x-search#1]
![71458d1843df43d020b4dd59f23186b](https://user-images.githubusercontent.com/18107069/216750236-0cfd1600-2225-4b35-8d7a-8f32f5c13749.png)

The aliases field may be null or a string，Suggested additional judgments

**Comment by Feng-YiJing-Dao** [comment]
You can start by converting the string to an array with a comma spacer, and then use the filter function

**Comment by Stardusten** [comment]
Can you provide some steps to reproduce yout problems?

**Comment by Stardusten** [comment]
This is an odd question for me because from my code, it's not likely to happen. This is what I wrote:

```ts
const aliases = frontMatter.aliases?.filter((a: any) => typeof a == 'string') ?? [];
```

**Comment by Stardusten** [comment]
Can you provide some steps to reproduce your issue? It's an odd question for me because I have checked the null condition for `aliases` and set the default value to `[]`.

This is what I wote:

```ts
const aliases = frontMatter.aliases?.filter((a: any) => typeof a == 'string') ?? [];
```

**Comment by Feng-YiJing-Dao** [comment]
This may be a problem on my side, my aliase field is a string when there is only one, not []

What I see in your code is 
```js
const aliases = (_d = (_c = frontMatter.aliases) == null ? void 0 : _c.filter((a2) => typeof a2 == "string")) != null ? _d : [];
```

I changed it to 
```js
const aliases = (_d = (_c = frontMatter.aliases) == null ? void 0 : typeof(_c) == "string"? Array.from(_c).filter((a2) => typeof a2 == "string"): _c.filter((a2) => typeof a2 == "string")) != null ? _d : [];
```
 to fit my situation

**Comment by Stardusten** [comment]
Should be fixed in the latest release 0.1.0.
